### ISSUE TITLE
Need assistance mapping gravity forms to salesforce

### ISSUE THREAD

**Issue by dsamwell** [katzwebservices/Gravity-Forms-Salesforce#287]
Hi,

We are looking for assistance mapping 6 gravity forms to salesforce and 1 Elegant themes bloom form.

We've done most of the work using the web-to-lead function but some of the forms aren't mapping and the rules for sending out the correct automated emails aren't working.

Is there anyone within this community that has experience setting this up and are you able to assist? If yes, what would your rates be?

Thanks in advance for your help

Cheers
Damien

**Comment by LaughterOnWater** [comment]
Have you checked out this issue number?
It's likely due to SF's upgrade to TLS1.1, and the need for the rest of us to upgrade our servers to work with the same, with regards to openssh.
https://github.com/katzwebservices/Gravity-Forms-Salesforce/issues/285

**Comment by dsamwell** [comment]
Hi,

Our hosting is with WPengine so will need to check that.

Are you saying that the updated SF only works with a certain Server setup? Sorry..Im a bit of a noob with SF/gravity forms integrations and we are having some annoying issues with mapping

Cheers
Damien

**Comment by LaughterOnWater** [comment]
Yes. If you're WP install is hosted on a typical linux box, you'll need to ask WPEngine to upgrade your box to use OpenSSL 1.0.1 for it to be TLS 1.1 compliant, according to the Salesforce link in the thread of issue 285.

**Comment by dsamwell** [comment]
Yep ive checked and WP engine meet all the criteria...hmm not to worry.

Just need an expert at salesforce/gravity forms mapping to help with this :(

thanks
Damien
